The travis for release-1.9 failed with the following error:


org.apache.kafka.common.KafkaException: Socket server failed to bind to 0.0.0.0:44867: Address already in use.
	at org.apache.flink.streaming.connectors.kafka.KafkaITCase.prepare(KafkaITCase.java:58)
Caused by: java.net.BindException: Address already in use
	at org.apache.flink.streaming.connectors.kafka.KafkaITCase.prepare(KafkaITCase.java:58)


instance:Â https://api.travis-ci.org/v3/job/629636116/log.txt